cmd/buckd/main.go

```diff
@@ -230,7 +230,7 @@ var rootCmd = &cobra.Command{
 		debug := config.Viper.GetBool("log.debug")
 		logFile := config.Viper.GetString("log.file")
 		if logFile != "" {
-			err = util.SetupDefaultLoggingConfig(logFile)
+			err = cmd.SetupDefaultLoggingConfig(logFile)
```

Comment:
Q: `SetupDefaultLoggingConfig` seems to have a fixed `ERROR` level. Is that correct considering the `log.debug` flag? Or maybe `log.debug` only applies if `log.file` is empty?

Reply:
This just sets the default on start, and then later on the debug flag is applied to textile modules. Setting anything other than debug here would cause all the libp2p modules to spew lots of logs.